server/json-schemas/noncommercial-fields-schema.es6

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  'id': '/noncommercialFields',
+  'type': 'object',
+  'properties': {
+    'activityDescription': {
+      'default':'',
+      'type': 'string'
+    },
+    'locationDescription': {
+      'default':'',
+      'type': 'string'
+    },
+    'startDateTime': {
+      'default':'',
+      'pattern':'^(19|20)\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3])Z$',
+      'type': 'string'
+    },
+    'endDateTime': {
+      'default':'',
+      'pattern':'^(19|20)\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3])Z$',
+      'type': 'string'
+    },
+    'numberParticipants': {
+      'default': '0',
+      'type': 'integer'
+    },
+    'formName':{
```

Comment:
Do you all really need to keep or submit the form number? The middle layer picks the form number based on it's logic. I think we're trying to shield you from it, but let me know if there's some reason to keep it here.